ntoskrnl/mm/balance.c

```diff
@@ -336,6 +343,15 @@ MiBalancerThread(PVOID Unused)
                     InitialTarget = MiTrimMemoryConsumer(i, InitialTarget);
                 }
 
+                /* Trim cache */
+                Target = max(InitialTarget, MiMinimumAvailablePages - MmAvailablePages);
+                if (Target)
+                {
+                    CcRosTrimCache(Target,
```

Comment:
```suggestion
                    CcRosTrimCache(Target, &NrFreedPages);
```
I would omit the linebreak here and put both parameters in the single line.
You have to accept both of my suggestions then, this is suggestion 1 of 2.